3.x/zh_CN/docs/tutorial/air/multihost.md

```diff
@@ -0,0 +1,348 @@
+# 搭建多机区块链网络
+
+标签：``搭建多机区块链网络`` ``区块链教程`` ``HelloWorld`` ``控制台调用合约``
+
+----
+
+[搭建第一个区块链网络](../../quick_start/air_installation.md)章节详细介绍了如何搭建Air版本单机4节点区块链网络，本章以搭建多机4节点区块链网络为例，详细介绍如何在多台机器上部署FISCO BCOS。
+
+## 1. 搭建多机4节点区块链网络
+
+本节详细介绍如何基于[部署工具(build_chain.sh)](./build_chain.md)搭建多机4节点区块链系统。
+
+本教程，假设四台物理机器的IP分别为`196.168.0.1`, `196.168.0.3`, `196.168.0.4`和`196.168.0.2`，每台机器部署一个区块链节点。
+
+```eval_rst
+.. note::
+    - 请确保每台机器的 ``30300``, ``20200`` 端口没有被占用。
+    - 请确保每台机器开通了 ``30300``, ``20200`` 端口的网络访问权限
+    - 请确保生成区块链节点配置的机器可以访问外网(用于下载建链脚本)
+```
+
+### 第一步. 下载部署工具并生成多机节点配置
+
+**创建操作路径，下载开发部署工具build_chain**
+
+```bash
+# 创建操作路径~/fisco
+mkdir -p ~/fisco && cd ~/fisco
+
+# 下载开发部署工具build_chain
+curl -#LO https://github.com/FISCO-BCOS/FISCO-BCOS/releases/download/v3.0.0-rc3/build_chain.sh && chmod u+x build_chain.sh
+```
+
+**生成多机节点配置**
+
+```bash
+bash build_chain.sh -l "196.168.0.1:1,196.168.0.2:1,196.168.0.3:1,196.168.0.4:1" -o 4nodes -e ./fisco-bcos
```

Comment:
Supplementary steps for downloading fisco-bcos.

Reply:
has added Supplementary steps